tests/antithesis/test-template/go-delete-keys/serial_driver_delete_keys.go

```diff
@@ -45,77 +49,64 @@ func Connect() *client.RecordingClient {
-	for _, k := range resp.Kvs {
-		keys = append(keys, string(k.Key))
-	}
-	half := len(keys) / 2
-	halfKeys := keys[:half]
+var etcdTraffic traffic.Traffic = traffic.EtcdTraffic{
+	KeyCount: 10,
+	// Please keep the sum of weights equal 100.
+	Requests: []robustnessrand.ChoiceWeight[traffic.EtcdRequestType]{
+		{Choice: traffic.Get, Weight: 50},
+		{Choice: traffic.Put, Weight: 50},
+	},
+}
 
-	// Connect to a new etcd node
-	cli = Connect()
+var profile = traffic.Profile{
+	MinimalQPS:                     100,
+	MaximalQPS:                     1000,
+	BurstableQPS:                   1000,
+	ClientCount:                    3,
+	MaxNonUniqueRequestConcurrency: 3,
+}
 
-	// Delete half of the keys chosen
-	var deletedKeys []string
-	for _, k := range halfKeys {
-		_, err := cli.Delete(ctx, k)
-		// Antithesis Assertion: sometimes delete requests are successful. A failed request is OK since we expect them to happen.
-		assert.Sometimes(err == nil, "Client can make successful delete requests", map[string]any{"error": err})
-		if err != nil {
-			log.Printf("Failed to delete key %s: %v", k, err)
-		} else {
-			log.Printf("Successfully deleted key %v", k)
-			deletedKeys = append(deletedKeys, k)
+// wrap converts a receive-only channel to receive-only struct{} channel
+func wrap[T any](from <-chan T) <-chan struct{} {
+	out := make(chan struct{})
+	go func() {
+		for {
+			<-from
+			out <- struct{}{}
 		}
-	}
-	cli.Close()
-
-	// Connect to a new etcd node
-	cli = Connect()
+	}()
+	return out
+}
 
-	// Check to see if those keys were deleted / exist
-	for _, k := range deletedKeys {
-		resp, err := cli.Get(ctx, k)
-		// Antithesis Assertion: sometimes get requests are successful. A failed request is OK since we expect them to happen.
-		assert.Sometimes(err == nil, "Client can make successful get requests", map[string]any{"error": err})
-		if err != nil {
-			log.Printf("Client failed to get key %s: %v", k, err)
-			continue
-		}
-		// Antithesis Assertion: if we deleted a key, we should not get a value
-		assert.Always(resp.Count == 0, "Key was deleted correctly", map[string]any{"key": k})
+func testRobustness() {
+	ctx := context.Background()
+	var wg sync.WaitGroup
+	var mux sync.Mutex
+	runfor := time.Duration(robustnessrand.RandRange(5, 60) * int64(time.Second))
+	limiter := rate.NewLimiter(rate.Limit(profile.MaximalQPS), profile.BurstableQPS)
+	finish := wrap(time.After(runfor))
+	reports := []report.ClientReport{}
+
+	for range profile.ClientCount {
+		wg.Add(1)
+		c := Connect()
+		go func(c *client.RecordingClient) {
+			defer wg.Done()
+			defer c.Close()
+
+			etcdTraffic.RunTrafficLoop(ctx, c, limiter,
+				identity.NewIDProvider(),
+				identity.NewLeaseIDStorage(),
+				traffic.NewConcurrencyLimiter(profile.MaxNonUniqueRequestConcurrency),
+				finish,
+			)
+			mux.Lock()
+			reports = append(reports, c.Report())
+			mux.Unlock()
+		}(c)
 	}
-	cli.Close()
-
-	assert.Reachable("Completion of a key deleting check", nil)
```

Comment:
Can you keep a assert like this, just change the text. Like `Completion of robustness traffic generation`.

Reply:
done.